docs/docsite/rst/network/user_guide/platform_eos.rst

```diff
@@ -27,30 +28,20 @@ Connections Available
 
     Connection Settings   ``ansible_connection: network_cli``         ``ansible_connection: httpapi``
 
-                                                                      OR
-
-                                                                      ``ansible_connection: local``
-                                                                      with ``transport: eapi``
-                                                                      in the ``provider`` dictionary
 
     |enable_mode|         supported: |br|                             supported: |br|
 
                           * use ``ansible_become: yes``               * ``httpapi``
                             with ``ansible_become_method: enable``      uses ``ansible_become: yes``
                                                                         with ``ansible_become_method: enable``
 
-                                                                      * ``local``
-                                                                        uses ``authorize: yes``
-                                                                        and ``auth_pass:``
-                                                                        in the ``provider`` dictionary
-
     Returned Data Format  ``stdout[0].``                              ``stdout[0].messages[0].``
     ====================  ==========================================  =========================
 
 .. |enable_mode| replace:: Enable Mode |br| (Privilege Escalation)
 
 
-For legacy playbooks, EOS still supports ``ansible_connection: local``. We recommend modernizing to use ``ansible_connection: network_cli`` or ``ansible_connection: httpapi`` as soon as possible.
+The ``ansible_connection: local`` has been deprecated. Please use ``ansible_connection: network_cli`` or ``ansible_connection: httpapi`` instead.
```

Comment:
Since task example use FQ collection reference, the connection plugin name has FQ reference. That is change `network_cli` to `ansible.netcommon.network_cli` similarly for httpapi